4-Express/server/models/todoModel.js

```diff
@@ -0,0 +1,41 @@
+let todos = []
```

Comment:
`const`

Reply:
i am changing the todo when deleting a todo from it - so it cant be const.